packages/matter.js/src/MatterDevice.ts

```diff
@@ -76,9 +76,14 @@ export class MatterDevice {
         private readonly discriminator: number,
         private readonly initialPasscode: number,
         private readonly storage: StorageContext,
-        private readonly initialCommissioningCallback: () => void,
+        private readonly commissioningChangedCallback: (fabricIndex: FabricIndex) => void,
+        private readonly sessionChangedCallback: (fabricIndex: FabricIndex) => void,
     ) {
-        this.fabricManager = new FabricManager(this.storage);
+        this.fabricManager = new FabricManager(this.storage, (fabricIndex: FabricIndex, peerNodeId: NodeId) => {
```

Comment:
Three of the four places we use the new second parameter just stub it.  I know they're just tests but given that the callback isn't used in the constructor WDYT about just leaving as a property that FabricManager can set?

Reply:
I just made it optional